docs/translation.md

```diff
@@ -54,12 +54,8 @@ changes to the ARB file.
 
 ### Using a translated string in the code
 
-To use in our widgets, you need to import the generated bindings:
-```
-import 'package:flutter_gen/gen_l10n/zulip_localizations.dart';
```

Comment:
super nit: the main migration commit should update this line, because otherwise the instruction is incorrect as of that commit

(then the later commit can delete, like in this revision, along with that nice commit message explaining why it used to be needed and no longer is)

Reply:
```diff
-import 'package:flutter_gen/gen_l10n/zulip_localizations.dart';
+import '/generated/l10n/zulip_localizations.dart';
```

If this nit were the only thing remaining in this revision I'd probably just go ahead and merge it, but since we have another round anyway: this (temporary) advice should match what we actually do in the source files :slightly_smiling_face:, which follows https://github.com/zulip/zulip-flutter/pull/1058#discussion_r1844485771 .